tests/conftest.py

```diff
@@ -140,6 +140,11 @@ def skip_if_no_tf_support(tf_support):
         pytest.skip("Skipped, no tf support")
 
 
+@pytest.fixture
+def skip_if_no_jax_support():
+    pytest.importorskip("jax")
+
+
```

Comment:
I like that this is simpler, but should we instead match the import pattern used for Torch and TF?

Reply:
The above were written when I did not know of the existence of `importorskip`. I think this is simpler, and happy for the above to be rewritten to match.